core/src/test/java/org/lflang/tests/LFParsingTest.java

```diff
@@ -33,6 +33,18 @@ public void testLexingEmptyTargetProperties() throws Exception {
     // assertNoParsingErrorsIn("target C {x:[,]};  \nreactor Foo {}");
   }
 
+  @Test
+  public void testParsingTargetPropertyAsString() throws Exception {
+    assertNoParsingErrorsIn("target C { \"a c time reactor\": 2 }; reactor Foo{}");
+    assertNoParsingErrorsIn("target C { \"reactor\": 2 }; reactor Foo{}");
+    expectParsingErrorIn("target C { reactor: 2 }; reactor Foo{}");
+
+    // array elements
```

Comment:
This commented-out code seems irrelevant. Remove?

Reply:
I see they're also there in `testLexingEmptyTargetProperties` :shrug:

Not a big deal, I guess. Let's just merge.